graph/doc.go

```diff
@@ -1,3 +1,3 @@
-// Package graph demonstates Graph search algorithms
+// Package graph demonstrates Graph search algorithms
```

Comment:
It would also be nice to add your authorship in these files 😄. Look at our contribution guidelines, it has an example of this.

```suggestion
// filename
// description: Add one line description here
// details:
// This is a multi-line
// description containing links, references,
// math equations, etc.
// author(s) [Name](https://github.com/handle), [Name](https://github.com/handle)
// see relatedfile.go, anotherfile.go, file_test.go
 
// ** Is just an example of how to write description for package and function and other stuff ... **

// Package sort provides primitives for sorting slices and user-defined
// collections.

// Package graph demonstrates Graph search algorithms
```

Reply:
At first, I thought of adding it, but *(no disrespect)* I didn't like its format.

I'll add it no problem.

Reply:
Ah ok, if you'd rather not that's fine – just thought that I should mention it. If you have a suggestion for better format, let us know in the issues if it is something that can be considered.

Reply:
How about a simple description just like as we do with the package description, along with an authorship comment?

```go
// This file contains the simple structural implementation of undirected
// graph. Used in coloring algorithms [here](./coloring)
// Author(s): [Shivam](https://github.com/Shivam010)

package graph
```

Reply:
As long as it is not the package level comment it should be fine. Package level comment (comment right above the package declaration without a '\n' character in between) is used to generate this repos automatic documentation in godocs so just keep that in mind😄.

Reply:
Yes, that's why I kept the line in my suggestion as well.